docs/src/main/paradox/java/http/server-side/index.md

```diff
@@ -0,0 +1 @@
+../../../scala/http/server-side/index.md
```

Comment:
yeah seems when I pulled this symlink did not get created somehow... I wonder why it fails the other PR but this one was fine